source/tutorials/qiime2-for-experienced-microbiome-researchers.rst

```diff
@@ -0,0 +1,296 @@
+
+.. _`Remove non-biological sequences`:
+
+Removing non-biological sequences
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your data contains any non-biological sequences (e.g. primers, sequencing adapters, PCR spacers, etc), you should remove these.
+
+The :doc:`q2-cutadapt <../plugins/available/cutadapt/index>` plugin has comprehensive methods for removing non-biological sequences from :doc:`paired-end <../plugins/available/cutadapt/trim-paired/>` or :doc:`single-end <../plugins/available/cutadapt/trim-single/>` data.
+
+If you're going to use DADA2 to denoise your sequences, you can remove biological sequences at the same time as you call the denoising function.
+All of DADA2's ``denoise`` fuctions have some sort of ``--p-trim`` parameter you can specify to remove base pairs from the end(s) of your reads.
+(Deblur does not have this functionality yet.)
+
+Grouping similar sequences
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are two main approaches for grouping similar sequences together: denoising and clustering.
+The overview tutorial provides more in-depth discussion of these approaches.
+
+Regardless of how you group your sequences, the grouping methods will output:
+
+1. A list of representative sequences for each of your OTUs and/or ASVs (QIIME 2 data format ``FeatureData[Sequence]``), and
+2. A feature table which indicates how many reads of each OTU/sequence variants were observed in each sample. (QIIME 2 data format ``FeatureTable[Frequency]``)
+
+DADA2 and deblur will also produce a stats summary file with useful information regarding the filtering and denoising.
+
+Denoising
+~~~~~~~~~
+
+DADA2 and deblur are currently the two denoising methods available in QIIME 2.
+You can read more about the methods in the overview tutorial.
+
+DADA2 and deblur both output exact sequence variants, which supposedly represent the true biological sequences present in your data.
+Their creators have different terminology for these sequences (DADA2 calls them "amplicon sequence variants" (ASVs) and deblur calls them "sub-OTUs").
+We'll be using the ASV terminology throughout this tutorial to refer to both outputs.
+
+Preparing data for denoising
+''''''''''''''''''''''''''''
+
+Denoising requires little data preparation.
+Both DADA2 and deblur perform quality filtering, denoising, and chimera removal, so you shouldn't need to perform any quality screening prior to running them.
+That said, the official QIIME 2 tutorial recommends doing an initial :ref:`quality-filter <moving pictures deblur>` with default settings prior to using deblur (as recommended by the deblur developers).
+In our experience, DADA2 performs better without this step.
+
+Both methods have an option to truncate your reads to a constant length (which occurs prior to denoising).
+The truncating parameter is required for deblur and optional for DADA2.
```

Comment:
that parameter can be disabled for deblur, so you can probably just drop this line.

Reply:
or at least that's what the docs suggest... I vaguely recall there being some mention on the forum that even though you can switch this off it leads to other issues with deblur:
```
  --p-trim-length INTEGER         Sequence trim length, specify -1 to disable
                                  trimming.  [required]
```

Reply:
mmk, if I add this sentence do you think that addresses the concerns?

> The truncating parameter is optional for both DADA2 and deblur (though if you're using deblur you'll need to specify `--p-trim-length -1` to disable truncation).

Reply:
👍